worker/src/lib.rs

```diff
@@ -135,6 +134,93 @@ pub async fn witness_tweet(
         };
         vc.evidence = Some(OneOrMany::One(evidence));
 
+        info!("{:?}", vc);
+
+        let proof = jserr!(
+            vc.generate_proof(
+                &sk,
+                &LinkedDataProofOptions {
+                    verification_method: Some(format!("{}#controller", SPRUCE_DIDWEB)),
+                    ..Default::default()
+                }
+            )
+            .await
+        );
+        vc.proof = Some(OneOrMany::One(proof));
+        Ok(jserr!(serde_json::to_string(&vc)).into())
+    })
+}
+
+#[wasm_bindgen]
+pub async fn witness_discord(
+    secret_key_jwk: String,
+    public_key_tezos: String,
+    discord_authorization_key: String,
+    discord_handle: String,
+    channel_id: String,
+    message_id: String,
+) -> Promise {
+    use log::Level;
+    console_log::init_with_level(Level::Trace).expect("error initializing log");
```

Comment:
Could you factor out the logging initialisation and put the level to Error so we don't have unimportant logs in production? (I don't even know if there's a way to get logs from live workers)